rust/Makefile

```diff
@@ -150,3 +150,7 @@ $(objtree)/rust/kernel.o: private rustc_target_flags = --extern alloc \
 $(objtree)/rust/kernel.o: $(srctree)/rust/kernel/lib.rs $(objtree)/rust/alloc.o \
     $(objtree)/rust/libmodule.so $(objtree)/rust/bindings_generated.rs FORCE
 	$(call if_changed_dep,rustc_library)
+
+GENERATE_RUST_ANALYZER = $(srctree)/scripts/generate_rust_analyzer.sh
```

Comment:
Do we need the variable, since it is only used once and people will not need to override it?

Reply:
My Idea was that this my create better output from Make, as it uses the `Q` thing? I don't understand `$(Q)` enough. So If you say remove it, then I can remove it.

Reply:
`Q` hides the output when not in verbose mode -- it is orthogonal to whether we use the variable or not.

I think you might be thinking about the "quiet" commands (i.e. the fancy `CC     file` output), but that requires defining a command and calling it. For something that will only be run when someone setups the development environment, I think it is fine to go simple.